src/components/Button/styles.js

```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from "react-native";
+
+/* eslint  import/prefer-default-export: 0 */
+export const styles = StyleSheet.create({
+  button: {
+    width: "100%",
+    borderRadius: 6,
+    paddingTop: 13,
+    paddingBottom: 13
+  },
+  buttonDark: {
+    backgroundColor: "#546e7a"
+  },
+  buttonLight: {
+    backgroundColor: "#ffffff",
+    borderWidth: 1,
+    borderColor: "#546e7a"
```

Comment:
Can we use constants for repeated colors/sizes/etc... ?

Reply:
Updated